src/components/CreateResetPassword/CreateResetPassword.js

```diff
@@ -0,0 +1,233 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { connect } from 'react-redux';
+import { reduxForm, Field, Form, formValueSelector } from 'redux-form';
+import { FormattedMessage } from 'react-intl';
+
+import _ from 'lodash';
+
+import { TextField, Button, Row, Col } from '@folio/stripes-components';
+
+import FieldLabel from './components/FieldLabel';
+import OrganizationLogo from '../OrganizationLogo';
+import AuthErrorsContainer from '../AuthErrorsContainer/AuthErrorsContainer';
+
+import styles from './CreateResetPassword.css';
+
+class CreateResetPassword extends Component {
+  static propTypes = {
+    stripes: PropTypes.shape({
+      intl: PropTypes.shape({
```

Comment:
Retrieving the intl object from this.context.intl or the stripes god object (stripes.intl) is a deprecated pattern.
Update it to only use react-intl constructs outlined at https://github.com/folio-org/stripes-core/blob/master/doc/i18n.md
Use <FormattedMessage>, <FormattedDate>, and <FormattedTime> as often as possible, only falling back to formatMessage(), formatDate(), and formatTime() (accompanied by injectIntl) when a bare string is absolutely necessary.

formatDateTime() should be replaced by <FormattedTime> or formatTime() with day/month/year props like https://github.com/folio-org/stripes-core/blob/d0d8f71ec322c9b89ae6ba92cb29179db1cc1704/util/dateUtil.js#L21

Util functions formatDate() and formatDateTime() defined in this repo shouldn't be necessary.

React Intl docs: https://github.com/yahoo/react-intl/wiki